docs/docs-content/integrations/community_packs.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_label: "Community Packs"
+title: "Community Packs"
+description: "Identify Community Packs in Palette, which are not actively maintenance."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["packs", "community"]
+---
+
+Palette Community Packs are packs that are created and maintained by the Spectro Cloud community. They provide important
+integrations with Spectro Cloud Palette.
+
+We welcome all contributions to the Spectro Cloud community packs, either by adding new packs or updating existing
+packs. You can read our
+[Contribution Guidelines](https://github.com/spectrocloud/pack-central?tab=readme-ov-file#contributing) in the community
+packs repository. Check out the [Deploy a Custom Pack](../registries-and-packs/deploy-pack.md) tutorial to learn more
+about how to build and deploy your own custom pack.
+
+Palette comes with a default community pack registry hosted by Spectro Cloud that is available to all SaaS tenants, and
```

Comment:
does SaaS include dedicated SaaS?

Reply:
Good question. Will figure it out.

Reply:
Dedicated SaaS environments need to add the community registry. I've changed the sentence to "multi-tenant SaaS".